Source/cryptalgo/SecureSocketPort.cpp

```diff
@@ -230,34 +262,44 @@ void SecureSocketPort::Handler::Update() {
 
         ASSERT(_ssl != nullptr);
 
+        // Client
         if (_handShaking == CONNECTING) {
             if ((result = SSL_connect(static_cast<SSL*>(_ssl))) == 1) {
-                _handShaking = EXCHANGE;
+                _handShaking = CONNECTED;
```

Comment:
I think the ValidateHandshake will set the _handShaking correctly based on the outcome, so no real need to set it if you call the ValidateHandShake() method, it will become CONNECTED or ERROR I assume.

Reply:
See https://github.com/rdkcentral/Thunder/pull/1796#discussion_r1858044490